modules/pool/lib/apr-data-sources/yb-apr-handlers/sources/sv-eth.ts

```diff
@@ -0,0 +1,40 @@
+import { data } from '../../../../../network/mainnet/data';
+import { createPublicClient, formatEther, http, parseAbi } from 'viem';
+import { mainnet } from 'viem/chains';
+import type { AprHandler } from '../';
+
+const client = createPublicClient({
+    chain: mainnet,
+    transport: http(data.rpcUrl),
```

Comment:
You anyway hardcode mainnet here as chain, could also just do `AllNetworkConfigs['MAINNET].rpcUrl`